Short-Answer/Algorithms_Answers.md

```diff
@@ -2,14 +2,41 @@
 
 ## Exercise I
 
-a)
+a) n is our input. If n gets bigger, this program will perform more operations (for loops). Since n is cubed on line 2, as our input gets bigger, the number of operations has a polynomial growth at a worst case scenario of O(n^3). 
 
 
-b)
+b) n is our input. If n gets bigger, this program will perform more operations (for loops). There is also a nested loop that depends on the size of n as well.  So the number of operations will have a polynomial growth at worst case scenario of O(n^2).
```

Comment:
Close on this one, but since j in the inner loop is doubling each time through rather than just counting up by 1, this would be O(nlogn)